src/OrchardCore.Modules/OrchardCore.Users/Views/Account/LinkExternalLogin.cshtml

```diff
@@ -7,7 +7,7 @@
 <form asp-controller="Account" asp-action="LinkExternalLogin" asp-route-returnurl="@ViewData["ReturnUrl"]" method="post" class="form-horizontal no-multisubmit">
     <h4>@T["Link your account."]</h4>
     <p class="text-info">
-        @T["You've successfully authenticated with <strong>{0}</strong>. You already have an account with this email address. Enter your local account password and click the Register button to link the accounts and finish logging in.", ViewData["LoginProvider"]]
+        @T["You've successfully authenticated with <strong>{0}</strong>. Seems already exist an account identificable by this information: <strong>{1}</strong>. Enter your local account password and click the Register button to link the accounts and finish logging in.", ViewData["LoginProvider"], ViewData["LinkParameterValue"]]
```

Comment:
The language should be improved here

Reply:
I try 😅 ... it's difficult for me to find a way to explain a generic concept in english